esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -1284,12 +1341,18 @@ def create_map_plot(self, datasets, short_name):
                 for netcdf_path in netcdf_paths:
                     provenance_logger.log(netcdf_path, provenance_record)
 
-    def create_profile_plot(self, datasets, short_name):
-        """Create profile plot."""
-        plot_type = 'profile'
-        if plot_type not in self.plots:
+    def create_zonal_mean_profile_plot(self, datasets, short_name):
+        """Create zonal mean profile plot."""
+        plot_type = 'zonal_mean_profile'
+        if plot_type not in self.plots and 'profile' not in self.plots:
             return
 
+        if 'profile' in self.plots:
+            logger.warning("Plot type 'profile' is deprecated and will be"
+                           " removed in version 2.10. It was renamed to"
+                           " 'zonal_mean_profile'")
+            self.plots['zonal_mean_profile'] = self.plots['profile']
+
```

Comment:
Can you move this part to the `__init__` function right below line 408 (`# Check given plot types and set default settings for them`)? This would simplify the remaining code a lot.

You can simply replace `profile` by `zonal_mean_profile` once in the beginning and then do not have to consider `profile` anywhere in the code.

E.g.,

```py
if 'profile' in self.plots:
    # show warning
    self.plots['zonal_mean_profile'] = self.plots.pop('profile')
```

Reply:
@schlunma Yes, that is a neat solution. Do you think it is a problem, that the options of `zonal_mean_profile` get overwritten by `profile` if both plotting types are used?

Reply:
Very good point! I think it's best if you raise an error when both options are used (you can check that right above this line), it doesn't make any sense to use both.